docs/examples/json_debbuging_reddit.py

```diff
@@ -0,0 +1,35 @@
+import json
```

Comment:
Added as a code comment. But if we figure out a nice PRAW debugging example (e.g., besides showing JSON, show some diagnostics or something like that), I'd be glad to add a tutorial and example in the future.